src/components/molecules/Results/Results.tsx

```diff
@@ -0,0 +1,18 @@
+import Ball from "../atom/Circle";
+
+import "./style.css";
+
+const Results = () => {
```

Comment:
Let's receive the results as an array and render all of them, it'll make your component flexible.